.github/workflows/test.yml

```diff
@@ -53,45 +53,18 @@ jobs:
       - name: Install yarn dependencies
         run: yarn install --frozen-lockfile
 
-      - name: Precompile using 0.8.19 and via-ir=false
+      - name: Precompile using 0.8.20 and via-ir=false
         run: yarn build
 
       - name: "Create env file"
         run: |
           touch .env
           echo OP_MAINNET_RPC="${{ secrets.OP_MAINNET_RPC }}" >> .env
-          echo OP_GOERLI_RPC="${{ secrets.OP_GOERLI_RPC }}" >> .env
+          echo OP_SEPOLIA_RPC="${{ secrets.OP_SEPOLIA_RPC }}" >> .env
           echo OP_MAINNET_DEPLOYER_PK="${{ secrets.TEST_DEPLOYER_PK }}" >> .env
-          echo OP_GOERLI_DEPLOYER_PK="${{ secrets.TEST_DEPLOYER_PK }}" >> .env
+          echo OP_SEPOLIA_DEPLOYER_PK="${{ secrets.TEST_DEPLOYER_PK }}" >> .env
           cat .env
 
       - name: Run tests
         shell: bash
-        run: yarn test
-
-  # coverage:
-  #   name: Run Coverage Tests
-  #   runs-on: ubuntu-latest
-
-  #   strategy:
-  #     matrix:
-  #       node-version: [16.x]
-
-  #   steps:
-  #     - uses: actions/checkout@v3
-  #     - name: Use Node.js
-  #       uses: actions/setup-node@v3
-  #       with:
-  #         node-version: ${{ matrix.node-version }}
-  #         cache: "yarn"
-  #     - name: Install Foundry
-  #       uses: foundry-rs/foundry-toolchain@v1
-  #       with:
-  #         version: nightly
-  #     - run: yarn install
-  #     - run: yarn build
-  #     - run: yarn coverage
-  #     - uses: codecov/codecov-action@v3
-  #       with:
-  #         files: ./coverage/lcov.info
-  #         flags: production
+        run: yarn test --nmt _Bytecode
```

Comment:
CI creates a different metadata, so let's skip it (yet it can be run locally)
also, still missing mainnet fork deployment to verify that everything is correct